tools/topology/topology2/include/components/src_format_host_capture.conf

```diff
@@ -0,0 +1,250 @@
+					out_rate                 48000
+					out_bit_depth            16
+					out_valid_bit_depth      16
+				}
+				{
+					out_rate                 48000
+					out_bit_depth            32
+					out_valid_bit_depth      24
+				}
+				{
+					out_rate                 48000
+					out_bit_depth            32
+					out_valid_bit_depth      32
+				}
+				# 64khz output
+				{
+					out_rate                 64000
+					out_bit_depth            16
+					out_valid_bit_depth      16
+				}
+				{
+					out_rate                 64000
+					out_bit_depth            32
+					out_valid_bit_depth      24
+				}
+				{
+					out_rate                 64000
+					out_bit_depth            32
+					out_valid_bit_depth      32
+				}
+				# 88.2khz output
+				{
+					out_rate                 88200
+					out_bit_depth            16
+					out_valid_bit_depth      16
+					obs "$[($out_channels * (($[($out_rate + 999)] / 1000) + 4)) * ($out_bit_depth / 8)]"
+				}
+				{
+					out_rate                 88200
+					out_bit_depth            32
+					out_valid_bit_depth      24
+					obs "$[($out_channels * (($[($out_rate + 999)] / 1000) + 4)) * ($out_bit_depth / 8)]"
+				}
+				{
+					out_rate                 88200
+					out_bit_depth            32
+					out_valid_bit_depth      32
+					obs "$[($out_channels * (($[($out_rate + 999)] / 1000) + 4)) * ($out_bit_depth / 8)]"
+				}
+				# 96khz output
+				{
+					out_rate                 96000
+					out_bit_depth            16
+					out_valid_bit_depth      16
+				}
+				{
+					out_rate                 96000
+					out_bit_depth            32
+					out_valid_bit_depth      24
+				}
+				{
+					out_rate                 96000
+					out_bit_depth            32
+					out_valid_bit_depth      32
+				}
+				# 176.4khz output
+				{
+					out_rate                 176400
+					out_bit_depth            16
+					out_valid_bit_depth      16
+					obs "$[($out_channels * (($[($out_rate + 999)] / 1000) + 4)) * ($out_bit_depth / 8)]"
+				}
+				{
+					out_rate                 176400
+					out_bit_depth            32
+					out_valid_bit_depth      24
+					obs "$[($out_channels * (($[($out_rate + 999)] / 1000) + 4)) * ($out_bit_depth / 8)]"
+				}
+				{
+					out_rate                 176400
+					out_bit_depth            32
+					out_valid_bit_depth      32
+					obs "$[($out_channels * (($[($out_rate + 999)] / 1000) + 4)) * ($out_bit_depth / 8)]"
+				}
+				# 192khz output
+				{
+					out_rate                 192000
+					out_bit_depth            16
+					out_valid_bit_depth      16
+				}
+				{
+					out_rate                 192000
+					out_bit_depth            32
+					out_valid_bit_depth      24
+				}
+				{
+					out_rate                 192000
+					out_bit_depth            32
+					out_valid_bit_depth      32
+				}
+			]
+
+			num_input_audio_formats 1
+
+			Object.Base.input_audio_format [
+				{
+					in_rate                 48000
+					in_bit_depth            32
+					in_valid_bit_depth      32
+				}
+			]
```

Comment:
also not clear why we all these output rates, but the inputs only have 48kHz. for the playback case where we are going to resample to 48kHz from 8..192kHz, is something missing?

Reply:
There is an existing include file for playback direction https://github.com/thesofproject/sof/blob/main/tools/topology/topology2/include/components/src_format.conf